### ISSUE TITLE
Update `Hangfire.Dashboard.Authorization` to target dotnet core

### ISSUE THREAD

**Issue by glen-84** [HangfireIO/Hangfire#2232]
Installing it on .NET 7, I get:

> /home/.../src/Api/Api.csproj : warning NU1701: Package 'Hangfire.Dashboard.Authorization 3.0.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net7.0'. This package may not be fully compatible with your project.
/home/.../src/Api/Api.csproj : warning NU1701: Package 'Owin 1.0.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework 'net7.0'. This package may not be fully compatible with your project.

**Comment by odinserj** [comment]
Yes, currently that package provides implementation only for .NET Framework, but not for the newer .NET platform versions. I will appreciate any help with this!
